crepecake/src/main/java/com/google/cloud/tools/crepecake/cache/CacheChecker.java

```diff
@@ -27,6 +28,7 @@
 import java.util.List;
 import java.util.Optional;
 
+// TODO: Put into CacheReader and remove this class.
```

Comment:
Filed https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/issues/121